recipes/polap-fmlrc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "polap-fmlrc" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "4463d4a71a8a3273108a0ab2dff0a4c37a565c05af6ee17ee83dc62404294caf" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/polap/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
```

Comment:
You include R packages as well, it's not a noarch python package, rather a generic one I think.

Reply:
This one does not use R package. I known it is confusing. I have other recipe that needs R packages.

Reply:
Actually, I wanted to have recipes only for linux-64 architecture because I have not tested my script with mac-osx. The script is intended to work with only Linux 64bit architecture because of its dependency and its data to deal with. I do not know that it is working with Mac OSX, which I guess not. Thank you.